### PR TITLE
Updating Shared Exchange docs

### PR DESCRIPTION
**Pull request by grimctl**

### CODE REVIEW

File: source/desktop/sharedexchange/sharedexdns.md [hunk 1]
```diff
@@ -33,10 +36,10 @@ Please allow time for any DNS changes to propagate through the internet, this ma
 
 * If you have any further questions please contact your dedicated account manager or speak with one of our support engineers on 0800 230 0032.
 
-```eval_rst
+``` eval_rst
```
Comment (terencesimms) [inline source/desktop/sharedexchange/sharedexdns.md]: need to remove this space

**Review by terencesimms — CHANGES_REQUESTED** [review]
I would change line 30 "(Please note: you must change “DOMAIN.COM” with your own domain name)" To a note, like;

``` note:: this is the note
```

File: source/desktop/sharedexchange/sharedexdns.md (hunk 1 above)
Comment (grimctl) [reply source/desktop/sharedexchange/sharedexdns.md]: Resolved in commit c82fbaa9501eecb13c9573fb3c466b4beb1dda18

**Comment by grimctl** [comment]
> I would change line 30 "(Please note: you must change “DOMAIN.COM” with your own domain name)" To a note, like;

Resolved in commit 5037700222dc26f675b4c76759c43fbb47664862

File: source/desktop/sharedexchange/sharedexdns.md [hunk 2]
```diff
@@ -9,22 +9,27 @@ If you would like to set these up manually please find our guide here, or contac
...
+| `hub4.ukfastexchange.co.uk` | `10` |
+| `client.ukfastexchange.co.uk` | `20` |
 
 Next, you should add an SRV record to the domain to allow for auto-discovery. This allows your Exchange ready client, such as Outlook, to find all the settings it needs to connect to our shared Exchange platform.
 
 Here is the information you need for the SRV record, in its raw format, should you not be using our UKFast SafeDNS system for DNS:
 
 `_autodiscover._tcp.DOMAIN.COM. 0 0 443 client.ukfastexchange.co.uk`
 
+```
```
Comment (terencesimms) [inline source/desktop/sharedexchange/sharedexdns.md]: The note needs to be like this
```
``` note:: you must change “DOMAIN.COM” with your own domain name.
```
with closing backticks which i apparently can't show here.

**Review by terencesimms — CHANGES_REQUESTED** [review]

File: source/desktop/sharedexchange/sharedexdns.md (hunk 2 above)
Comment (grimctl) [reply source/desktop/sharedexchange/sharedexdns.md]: Resolved in f84ee4f

**Review by terencesimms — APPROVED** [review]

**Pull request merged**